app/orchestrators/CorrectReturnOrchestrator.scala

```diff
@@ -48,7 +48,8 @@ class CorrectReturnOrchestrator @Inject()(connector: SoftDrinksIndustryLevyConne
 
     for {
       sdilReturn <- getSdilReturn(retrievedSubscription, selectedReturnPeriod)
-      updatedUserAnswers <- generateUserAnswersWithSdilReturn(userAnswers, sdilReturn, selectedReturnPeriod)
+      updatedUserAnswersWithSavedSdilReturn <- generateUAsWithSavedSdilReturn(userAnswers, sdilReturn)
+      updatedUserAnswers <- generateUserAnswersWithSdilReturn(updatedUserAnswersWithSavedSdilReturn, sdilReturn, selectedReturnPeriod)
```

Comment:
Is this function used multiple times? Otherwise could add savedSdilReturn functionality into that?